pytorch_lightning/trainer/training_tricks.py

```diff
@@ -261,7 +262,7 @@ def _adjust_batch_size(trainer,
                        batch_arg_name: str = 'batch_size',
                        factor: float = 1.0,
                        value: Optional[int] = None,
-                       desc: str = None):
+                       desc: str = None) -> Tuple[int, bool]:
     """ Function for adjusting the batch size. It is expected that the user
         has provided a model that has a hparam field called `batch_size` i.e.
         `model.hparams.batch_size` should exist. Additionally there can be a
```

Comment:
The description of this function seems outdated.

Reply:
Ok, since this is a (protected) helper function, I moved the detailed description to the main function scale_batch_size, and added info in which location we look for the fields. Anything else that needs be updated?